discovery/seata-discovery-redis/src/main/java/io/seata/discovery/registry/redis/RedisRegistryServiceImpl.java

```diff
@@ -246,6 +246,8 @@ List<InetSocketAddress> lookupByCluster(String clusterName) {
                         break;
                     case RedisListener.UN_REGISTER:
                         removeServerAddressByPushEmptyProtection(clusterName, serverAddr);
+
+                        removeOfflineAddressesIfNecessary(clusterName, CLUSTER_ADDRESS_MAP.get(clusterName));
```

Comment:
`removeOfflineAddressesIfNecessary` 方法 为什么不放到`removeServerAddressByPushEmptyProtection`方法中调用？这么写会导致 `CLUSTER_ADDRESS_MAP.get(clusterName) `被执行2次
`removeOfflineAddressesIfNecessary` method, why not call it in the `removeServerAddressByPushEmptyProtection` method? This will cause CLUSTER_ADDRESS_MAP.get(clusterName) to be executed twice

Reply:
done